src/EPR.Payment.Service/Services/PaymentsService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using AutoMapper;
+using EPR.Payment.Service.Common.Data.Interfaces.Repositories;
```

Comment:
Replace the file with the one I had sent to include cancellationtokens

Reply:
resolved